Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:8
```

Comment:
Your final Docker image size is 939MB, which is pretty large for the type of application you're running. Consider using a more lightweight Docker image. You can look at small linux distributions as your base image that will get the job done equally well like `alpine`, `slim` etc. You should be able to get your image down to less than 150MB. 

Be sure to read up on what is included in these base images to ensure you're picking one that has all the things you need to run your app.